# simple message for grasp shape
# weight
string grasp_shape

# time estimation
time time_now